.github/renovate.json

```diff
@@ -0,0 +1,13 @@
+{
+  "dependencyDashboard": true,
```

Comment:
If this is enabled, Renovate will maintain a GitHub issue about the status of various dependency updates, it can also be used to control its behavior via checkboxes. Example:

* https://github.com/typeddjango/djangorestframework-stubs/issues/868